test/jdk/java/util/Formatter/Basic-X.java.template

```diff
@@ -1697,6 +1600,16 @@ public class Basic$Type$ extends Basic {
 
         // Get the supported ids for GMT-08:00 (Pacific Standard Time)
         String[] ids = TimeZone.getAvailableIDs(-8 * 60 * 60 * 1000);
+        // With tzdata2020b, the test fails for the mentioned zones expecting
+        // "PST" but JDK has Zone name "MST" for JRE locale provider.
+        // Therefore excluding them as the test is only looking for a GMT-08:00
+        // time zone ID. See JDK-8254865.
+        final List<String> list =  new ArrayList<String>();
+        Collections.addAll(list, ids);
+        list.remove("America/Dawson");
+        list.remove("America/WhiteHorse");
+        list.remove("Canada/Yukon");
+        ids = list.toArray(new String[list.size()]);
```

Comment:
It's not your change but I just noticed this. Since we specifically test only for `CLDR` provider (as in `BasicTestLauncher`), I wonder we could remove this portion. Even if we do need this piece, it could be simplified as:
```
ids = Arrays.stream(ids).filter(tz -> !tz.equals(...)).toArray(String[]::new)
```

Reply:
Removing portion since it is no longer used/needed. Tests passing without.